examples/deploy/terraform/cluster/main.tf

```diff
@@ -79,6 +86,9 @@ module "irsa_external_deployments_operator" {
   use_cluster_odc_idp           = local.is_eks_account_same
   eks_info                      = module.eks.info
   external_deployments_operator = var.irsa_external_deployments_operator
+  region                        = local.region
```

Comment:
If this fields are needed by `external_deployments_operator` they should be contained in its definition and not be a requirement by the whole module.
see [this](https://github.com/dominodatalab/terraform-aws-eks/blob/main/modules/irsa/variables.tf#L105) example of the variable being passed